.github/workflows/ci.yml

```diff
@@ -20,12 +14,9 @@ jobs:
       - name: Setup Python
         uses: actions/setup-python@v2
         with:
-          python-version: ${{ matrix.python-version }}
+          python-version: "3.8"
 
-      - name: Run black
+      - name: Run lint pipelines
         run: |
           pip install nox
-          nox -s format
-
-      - name: Run mypy
-        run: nox -s mypy
+          nox -s format mypy
```

Comment:
Ideally I'd like to see these two steps staying separate.

Reply:
Split them into 2 workflows, since before, if black failed, mypy would never get to run, making it a bit harder to know what someone needs to fix. Hopefully this will make it easier